test/suite.js

```diff
@@ -38,6 +38,18 @@ describe('Helpers', function () {
       return merged.prop1.should.equal('red');
 
     });
+
+    it('should allow falsy child values to take precedence', function () {
+      var parentObject = {
+        count: 1
+      };
+      var childObject = {
+        count: 0
+      };
+      var merged = helpers.mergeJson(parentObject, childObject);
+
+      should.strictEqual(merged.count, 0);
```

Comment:
For some reason `merged.count.should.strictEqual(0)` failed with `undefined is not a function` suggesting strictEqual was not available in the should object.